portafly/portafly_api.yml

```diff
@@ -0,0 +1,102 @@
+---
+openapi: 3.0.2
+info:
+  title: PortaFly
+  version: 1.0.0
+  description: |
+    A tuned API for PortaFly, a Patternfly SPA consuming 3scale Porta API MGMT solution.
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /developer/accounts:
+    summary: A list of developer accounts
+    description: A list of developer accounts.
+    get:
+      responses:
+        "200":
+          $ref: '#/components/responses/developerAccountList'
+      operationId: developerAccountList
+      summary: Get developer account list
+    parameters:
+    - name: page
+      description: Page in the paginated list. Defaults to 1.
+      schema:
+        type: integer
+      in: query
+      required: false
+    - name: per_page
+      description: Number of results per page. Default and max is 500.
+      schema:
+        type: integer
+      in: query
+      required: false
+    - name: state
+      description: Account state.
+      schema:
+        type: string
+      in: query
+components:
+  schemas:
+    developerAccountData:
+      title: Root Type for developerAccountData
+      description: A simplified format for a developer account data.
+      type: object
+      properties:
+        id:
+          description: The account ID number
+          type: integer
+        created_at:
+          format: date-time
+          description: When it was created
+          type: string
+        updated_at:
+          format: date-time
+          description: When it was last updated
+          type: string
+        state:
+          description: The current state of the account
+          type: string
+        org_name:
+          description: The organization name
+          type: string
+        admin_name:
+          description: The name of the admin user
```

Comment:
Are you sure you'll only need the name of the user? Maybe a link to the user instead?

Reply:
Given the design in https://issues.redhat.com/browse/THREESCALE-4725, only the name is needed. No link for the admin user. It might be needed for the organization link that links to the account, which is not yet created. I thought about expanding the spec as we are building new resources. Does that make sense?

Reply:
I guess it makes sense, yes.